src/boot/boot_main.c

```diff
@@ -1,5 +1,6 @@
 #include "global.h"
 #include "boot.h"
+#include "cic6105.h"
```

Comment:
do we have to `#if` this out? (FWIW I don't like that practice anyway and prefer the way it is now)

Reply:
I prefer to, it's not a header that exists outside of N64 versions, and isn't used then. #if ing is also self-documenting. And I think reading the source for a version should be only "that version" stuff
Plus in the so-far-theoretical case you partially cpp the source to get source for a single version, you'd want those includes gone